### ISSUE TITLE
geocode.arcgis.com Bad HTTP Request 

### ISSUE THREAD

**Issue by TheAwesomeGcdm** [vbguyny/ws4kp#82]
When trying to choose a location when hosting on a local server, I get a bad HTTP request, even though I can access the host on another tab.

Browser: Firefox
OS: Windows 11

![imagen](https://user-images.githubusercontent.com/40742341/198842871-eb8e900d-aab0-49a6-b47d-08bfe99eb230.png)

**Comment by vbguyny** [comment]
Looks like a Cross-Origin Resource Sharing (CORS) issue.  This is a setting which is defined by the host and enforced by the browser.  There really isn't much you can do to work around this and it is a security feature of the web.
That being said you could try adding a hosts record that redirects you back to localhost.  You might also need to generate a self-signed SSL certificate and assign it the domain related to your local web server.
The best option would be to have your site hosted somewhere and it should work.

**Comment by TheAwesomeGcdm** [comment]
Okay, I followed your advice and created a SSL certificate for the local host by running the following command in PowerShell.
`New-SelfSignedCertificate -DnsName "localhost" -CertStoreLocation "cert:\LocalMachine\My"`

Once I did that, I went to Microsoft Management Console (run mmc.exe in the run window) and went to File -> Add or Remove Snap-ins -> Certificates -> Add -> Computer account -> Local computer

Under the Personal folder, I copied the localhost certificate there and pasted it in the Trusted Root Certification Authorities, closing the window and saving the changes. 

In IIS, I went to the page where WeatherStar is and went to Bindings -> Edit -> and changed http to https and set the SSL certificate to localhost.

Then I went to https://localhost:XXX/ws4kp and it works perfect!
